csrc/torch_memory_saver.cpp

```diff
@@ -210,60 +260,34 @@ class TorchMemorySaver {
             std::cout << "[torch_memory_saver.cpp] TorchMemorySaver.resume"
                       << " ptr=" << ptr << " metadata.size=" << metadata.size << " (old)metadata.allocHandle="
                       << metadata.allocHandle
-                      << " (new)newAllocHandle=" << newAllocHandle
+                      << " (new)newAllocHandle=" << newAllocHandle << " tag=" << metadata.tag << " filter_tag=" << tag
                       << std::endl;
 #endif
 
             metadata.allocHandle = newAllocHandle;
         }
     }
 
-    static TorchMemorySaver &instance() {
-        static TorchMemorySaver instance;
-        return instance;
-    }
```

Comment:
nit: maybe keep this unchanged (not renamed + not moved) to make code diff minimized